### PR TITLE
GitHub fixes

### PR DESCRIPTION
**Pull request by avarabyeu**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/user/impl/DeleteUserHandler.java [hunk 1]
```diff
@@ -62,26 +66,24 @@ public OperationCompletionRS deleteUser(String userId, String principal) {
...
 		OperationCompletionRS response = new OperationCompletionRS();
-		StringBuilder message = new StringBuilder();
-		message.append("User with ID = '");
-		message.append(userId);
-		message.append("' successfully deleted.");
+		String message = "User with ID = '" + userId + "' successfully deleted.";
 
-		response.setResultMessage(message.toString());
+		response.setResultMessage(message);
 		return response;
```
Comment (DzmitryKavalets) [inline src/main/java/com/epam/ta/reportportal/core/user/impl/DeleteUserHandler.java]: OperationCompletionRS has constructor with a String parameter

File: src/main/java/com/epam/ta/reportportal/core/project/impl/DeleteProjectHandler.java [hunk 2]
```diff
@@ -68,6 +69,7 @@ public OperationCompletionRS deleteProject(String projectName) {
 
 		Project project = projectRepository.findOne(projectName);
 		expect(project, notNull()).verify(PROJECT_NOT_FOUND, projectName);
+		expect(project.getConfiguration().getEntryType(), not(equalTo(EntryType.PERSONAL))).verify(UNABLE_TO_UPDATE_DEFAULT_PROJECT);
```
Comment (DzmitryKavalets) [inline src/main/java/com/epam/ta/reportportal/core/project/impl/DeleteProjectHandler.java]: Fix error message

Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/project/impl/DeleteProjectHandler.java]: Error message is correct: Unable to update or delete DEFAULT project of Report Portal. {}"

File: src/main/java/com/epam/ta/reportportal/core/user/impl/DeleteUserHandler.java (hunk 1 above)
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/user/impl/DeleteUserHandler.java]: Agreed

**Pull request merged**